python/ray/_private/worker.py

```diff
@@ -1277,6 +1277,7 @@ def init(
             and the API is subject to change.
         _node_name: User-provided node name or identifier. Defaults to
             the node IP address.
+        _enable_cluster_auth: Whether to enable cluster authentication.
```

Comment:
Can we just use a normal ray config var (ray_config_def.h)?

Reply:
ah...

Reply:
done